mgmtsystem/hawkular.py

```diff
@@ -0,0 +1,77 @@
+from collections import namedtuple
+from rest_client import ContainerClient
+
+"""
+Related yaml structures:
+
+[cfme_data]
+management_systems:
+    hawkular:
+        name: My hawkular
+        type: hawkular
+        hostname: 10.12.13.14
+        port: 8080
+        credentials: hawkular
+        authenticate: true
+        rest_protocol: http
+
+[credentials]
+hawkular:
+    username: admin
+    password: secret
+"""
+
+Feed = namedtuple('Feed', ['id', 'name', 'path'])
+ResourceType = namedtuple('Feed', ['id', 'name', 'path'])
+Server = namedtuple('Feed', ['id', 'name', 'path'])
+
+
+class Hawkular(object):
+
```

Comment:
This would also deserve a comment like in https://github.com/RedHatQE/mgmtsystem/blob/master/mgmtsystem/openstack.py#L55 .